### PR TITLE
added editor gamma options #2

### PR DESCRIPTION
**Pull request by thmasn**
#9230 
i hope i am doing this correctly

### CODE REVIEW

File: editor/js/libs/app.js [hunk 1]
```diff
@@ -23,10 +23,11 @@ var APP = {
 		this.load = function ( json ) {
 
 			vr = json.project.vr;
-
 			renderer = new THREE.WebGLRenderer( { antialias: true } );
 			renderer.setClearColor( 0x000000 );
 			renderer.setPixelRatio( window.devicePixelRatio );
+			renderer.gammaInput = json.project.gammaInput;
+			renderer.gammaOutput = true;
```
Comment (mrdoob) [inline editor/js/libs/app.js]: Shouldn't this be `renderer.gammaOutput = json.project.gammaOutput;`?

**Comment by thmasn** [comment]
yes. it is now

do the gamma setiings affect all renderer types or do i need to add conditions on when to display / apply them?

**Comment by mrdoob** [comment]
> do the gamma setiings affect all renderer types or do i need to add conditions on when to display / apply them?

It should be fine like this for now 😊

**Comment by mrdoob** [comment]
Thanks!

**Comment by thmasn** [comment]
nice

**Pull request merged**